pkg/receive/hashring.go

```diff
@@ -39,23 +36,6 @@ type Hashring interface {
 	GetN(tenant string, timeSeries *prompb.TimeSeries, n uint64) (string, error)
 }
 
-// hash returns a hash for the given tenant and time series.
-func hash(tenant string, ts *prompb.TimeSeries) uint64 {
-	// Sort labelset to ensure a stable hash.
-	sort.Slice(ts.Labels, func(i, j int) bool { return ts.Labels[i].Name < ts.Labels[j].Name })
```

Comment:
And the labels are not sorted, is this also intentional?

Reply:
Good point, I think tests I failing due this potentially as well